README.md

```diff
@@ -57,8 +57,55 @@ jobs:
         parameters: storageAccountType=Standard_LRS
 ```
 
-## Another example on how to use this Action to use get the output of ARM template
-In this exmaple, our template outputs `containerName`.
+## Another example which ensures the Azure Resource Group exists before ARM deployment
+In the preceeding example there is a pre-requisite that an existing Azure Resource Group named ```github-action-arm-rg``` must already exist.  
+
+The below example makes use of the [Azure CLI Action](https://github.com/marketplace/actions/azure-cli-action) to ensure the resource group is created before doing an ARM deployment.
+
+## Steps
+When generating your credentials (in this example we store in a secret named ```AZURE_CREDENTIALS```) you will need to specify a scope at the subscription level.
+
+```azurecli
+az ad sp create-for-rbac --name "{sp-name}" --sdk-auth --role contributor --scopes /subscriptions/{subscription-id}
+```
+
+See [Configure deployment credentials](https://github.com/marketplace/actions/azure-login#configure-deployment-credentials).
+
+## Example
+```yaml
+on: [push]
+name: AzureLoginSample
```

Comment:
Lets update this to. "AzureARMSample"

Reply:
@pulkitaggarwl done.

Reply:
[commit](https://github.com/Azure/arm-deploy/pull/34/commits)